src/core_write.cpp

```diff
@@ -114,6 +114,10 @@ string ScriptToAsmStr(const CScript& script, const bool fAttemptSighashDecode)
         }
 
         if (opcode == opcodetype::OP_ZEROCOINSPENDV3 || opcode == opcodetype::OP_ZEROCOINMINTV3) {
+            vch.clear();
+            vch.assign(pc, script.end());
```

Comment:
I think we should fix `GetOp` at line 85 to populate it with spend payload instead.